doc/aqd.rst

```diff
@@ -1,13 +1,17 @@
 Processing Aquadopp (currents) data
 ***********************************
 
-Data will generally be processed using a series of run scripts. The first script for each instrument type
-depends on two :doc:`configuration files </config>`.
+After getting the code, make sure you change sys.path or sys.path.append lines point to the equivalent place on the local system.
 
-Instrument data to raw .cdf
+** NOTE: this code works with up- or down-looking Aquadopp data collected in 'beam', 'XYZ' or 'earth'.  It doesn't work on HR AQD yet. **
+
+Data will generally be processed using a series of run scripts that use command line arguments.  FOr Aqd current's it's a 2 step process. 
+
+Step 1 : Instrument data to raw .cdf
 ===========================
 
-Convert from text to a raw netCDF file with ``.cdf`` extension.
+Step 1: Convert from text to a raw netCDF file with ``.cdf`` extension using runaqdhdr2cdf.py. This script 
+depends on two arguments, the global attribute file and extra configuration information :doc:`configuration files </config>`.
```

Comment:
Delete "configuration files" and move "configuration information" into its place, like so:

    :doc:`configuration information </config>`